### ISSUE TITLE
Storage Object List Fields

### ISSUE THREAD

**Issue by ghc4** [oracle/oci-dotnet-sdk#97]
Hi. 
I'm trying to list objects using dotnet sdk.
I need the md5 and the date of the object.
The "Fields" on the ObjectListRequest may do the job, but it is a single Enum. 
How can I get all fields at same time?

Enum code from ListObjectsRequest.cs 

<pre>
public enum FieldsEnum {
            [EnumMember(Value = "name")]
            Name,
            [EnumMember(Value = "size")]
            Size,
            [EnumMember(Value = "etag")]
            Etag,
            [EnumMember(Value = "timeCreated")]
            TimeCreated,
            [EnumMember(Value = "md5")]
            Md5,
            [EnumMember(Value = "timeModified")]
            TimeModified,
            [EnumMember(Value = "storageTier")]
            StorageTier,
            [EnumMember(Value = "archivalState")]
            ArchivalState
        };
</pre>

Fields property from same file

<pre>
        /// <value>
        /// Object summary by default includes only the 'name' field. Use this parameter to also
        /// include 'size' (object size in bytes), 'etag', 'md5', 'timeCreated' (object creation date and time),
        /// 'timeModified' (object modification date and time), 'storageTier' and 'archivalState' fields.
        /// Specify the value of this parameter as a comma-separated, case-insensitive list of those field names. 
        /// For example 'name,etag,timeCreated,md5,timeModified,storageTier,archivalState'.
        /// 
        /// </value>
        [Oci.Common.Http.HttpConverter(Oci.Common.Http.TargetEnum.Query, "fields")]
        public System.Nullable<FieldsEnum> Fields { get; set; }
</pre>

**Comment by github-anurag** [comment]
Hi @ghc4 
This is a bug in the SDK and will be addressed in the next release on Jan 11.

**Comment by ghc4** [comment]
Same problem on oci-dotnet-sdk/Objectstorage/requests/ListObjectVersionsRequest.cs

**Comment by github-anurag** [comment]
@ghc4 
Thank you for testing the SDK in depth. The release on Jan 11 will fix the issue in the following Requests:-
```
ListObjectVersionsRequest
ListObjectsRequest
ListIdentityProvidersRequest
DbNodeActionRequest
InstanceActionRequest
```

**Comment by github-anurag** [comment]
@ghc4 
This issue should now be fixed in the release 30.0.0. Please feel free to reopen this if the issue comes up again.
